opensrp-chw/src/crvs/assets/rule/death_certification_relevance.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: step1_informant_fields
+description: informant_fields
+priority: 1
+condition: "step1_received_death_certificate('Yes') || step1_death_notification_done('Yes')"
```

Comment:
This appears to be missing the comparison operation `.equals()` which breaks the relevance for the `informant_fields` step in the form. 
The label currently doesn't show on the form. Please fix this

Reply:
Noted.